pkg/node/node.go

```diff
@@ -188,15 +206,18 @@ func NewNode(
 			return nil, err
 		}
 
-		// TODO calling `Keys` on the publishers takes ~10 seconds per call
-		// https://github.com/bacalhau-project/bacalhau/issues/4153
-		metrics.NodeInfo.Add(ctx, 1,
-			attribute.StringSlice("node_publishers", publishers.Keys(ctx)),
-			attribute.StringSlice("node_storages", storages.Keys(ctx)),
-			attribute.StringSlice("node_engines", executors.Keys(ctx)),
-		)
+		/*
+			// TODO calling `Keys` on the publishers takes ~10 seconds per call
+			// https://github.com/bacalhau-project/bacalhau/issues/4153
+			metrics.NodeInfo.Add(ctx, 1,
+				attribute.StringSlice("node_publishers", publishers.Keys(ctx)),
+				attribute.StringSlice("node_storages", storages.Keys(ctx)),
+				attribute.StringSlice("node_engines", executors.Keys(ctx)),
+			)
+
+		*/
 
```

Comment:
why commenting this out?

Reply:
See the TODO above, this code takes ~10 seconds to run because we lack a way (that I am aware of) to disable bacalhau from attempting to retrieve s3 credentials for storages and publishers.